src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
```

Comment:
The function name 'convertToObject' is acceptable, but consider using a more descriptive name that reflects the conversion from a CSS string to an object.